### PR TITLE
Migration code tweaks

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

Belated tweaks to migrators as I wasn't able to review PR #4400 in time.

## How was this change tested? 🤨

unit tests.

⚡ ⚠ If this change has cross service impact, including data writes to shared file systems, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: bin/clean-druid-list [hunk 1]
```diff
@@ -1,34 +0,0 @@
-#!/usr/bin/env ruby
```
Comment (justinlittman) [inline bin/clean-druid-list:1]: I wouldn't get rid of this. Still a useful utility.

File: bin/generate-druid-list [hunk 2]
```diff
@@ -1,44 +0,0 @@
-#!/usr/bin/env ruby
```
Comment (justinlittman) [inline bin/generate-druid-list:1]: I wouldn't get rid of this. Still a useful utility.

File: app/services/migrators/invalid_lane_catkey.rb [hunk 3]
```diff
@@ -22,21 +22,21 @@ def version?
     end
 
     def version_description
-      'Invalidate Lane catkeys'
+      'Change Invalid Lane catkey to "previous symphony" identifier'
```
Comment (justinlittman) [inline app/services/migrators/invalid_lane_catkey.rb:25]: Did you check with Arcadia on this change?

**Review by justinlittman — CHANGES_REQUESTED** [review]
There is also a tweak needed in README.

File: bin/clean-druid-list (hunk 1 above)
Comment (ndushay) [reply bin/clean-druid-list:1]: whoops - mistake.

File: bin/generate-druid-list (hunk 2 above)
Comment (ndushay) [reply bin/generate-druid-list:1]: whoops mistake

File: app/services/migrators/invalid_lane_catkey.rb (hunk 3 above)
Comment (ndushay) [reply app/services/migrators/invalid_lane_catkey.rb:25]: I have now:

image

**Comment by ndushay** [comment]
@justinlittman thanks for feedback.  I think I've addressed it all.

**Pull request merged**